main.go

```diff
@@ -4,22 +4,26 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"github.com/antonlindstrom/mesos_stats"
-	"github.com/golang/glog"
-	"github.com/prometheus/client_golang/prometheus"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/antonlindstrom/mesos_stats"
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
 	configFile  = flag.String("config.file", "", "Path to config file.")
 	metricsPath = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+)
+
+var (
 	cpuLimitVal = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "mesos_task",
-			Name:      "cpus_limit",
+			Name:      "cpu_limit",
 			Help:      "CPU share limit.",
```

Comment:
Actually, I might have been way off. I've re-read the code and it should be the number of CPUs that is reported. However, this can be a fraction and it's possible to allocate something like `0.5` CPUs. So number of allowed cores may be more appropriate. Sorry for the confusion.